### PR TITLE
fix: show deprecated error for alias

### PR DESCRIPTION
**Pull request by a-tarasyuk**
Fixes https://github.com/microsoft/TypeScript/issues/40588#issuecomment-700386436

### CODE REVIEW

File: src/compiler/checker.ts [hunk 1]
```diff
@@ -28307,6 +28305,13 @@ namespace ts {
             return returnType;
         }
 
+        function checkDeprecatedIdentifier(node: Identifier, symbol: Symbol) {
```
Comment (sandersn) [inline src/compiler/checker.ts:28308]: I'd rather keep this inline since it's only used once and isn't that long

**Review by sandersn — CHANGES_REQUESTED** [review]
This doesn't fix the suggestions, does it? It looks like it's just the suggestions.

Also one style request.

**Comment by a-tarasyuk** [comment]
@sandersn Yes, this is a fix for the issue described in the https://github.com/microsoft/TypeScript/issues/40588#issuecomment-700386436.

**Review by sandersn — APPROVED** [review]

**Pull request merged**